docs/Course Wiki/Taking X with Y/ECE210 with ECE220.md

```diff
@@ -0,0 +1,19 @@
+# ECE210 with ECE220
+
+## Relevant Course Reviews:
+- [ECE210](../ECE%20Course%20Offerings/ECE210.md)
+- [ECE220](../ECE%20Course%20Offerings/ECE220.md)
+
+## Who should take them?
+Both courses are required for ECE majors. CompE majors typically do ECE220 in Freshman Spring and then ECE210 in Sopohmore year, while EE majors typically do both ECE210 and ECE220 sometime in Freshman Spring or Sophomore Fall semesters. Both courses are prerequisites for many of the 300/400-level ECE courses, and ECE220 is a prerequisite for [CS225](../CS%20Course%20Offerings/CS225.md). If you wish to take higher level ECE courses earlier, getting these two courses out of the way early on is key. It is also worth noting that both these courses are required for junior eligibility status. ECE210 focuses on circuit analysis concepts and basics of signal processing such as frequency response, Fourier transforms, and convolution, whereas ECE220 focuses on assembly language features such as stacks and subroutines, and programming basics such as Functions, Arrays, Pointers, Recursion, Simple Sorting Algorithms, and Basic Data Structures in C (and a brief overview on C++ in the final weeks of the course).
+
+## Workload:
```

Comment:
remove : after headings

i would focus this more about how the workload of the two classes intersect (hard at the same time etc) since this info is duplicated from class pages and will be harder to maintain